### PR TITLE
ui: fix seed backup page display

### LINKED ISSUE #72: trying to exit wallet back up process doesn't work
clicking on the yes button does nothing

https://github.com/crypto-power/cryptopower/assets/57448127/3da43f49-fe4a-49ac-815b-5ec42bc55afe

Screenshot 2023-09-20 at 6 08 23 AM

### PR DESCRIPTION
**Pull request by ukane-philemon**
Closes #72 

- Fix cancel seed backup that prevented the actual parent window display.
- Fix the password modal cancel bug on the view seed page

https://github.com/crypto-power/cryptopower/assets/57448127/49e6ae8c-7fdb-4c5a-a1a8-f42f2ec73926

### CODE REVIEW

File: ui/page/seedbackup/verify_seed.go [hunk 1]
```diff
@@ -261,7 +263,7 @@ func (pg *VerifySeedPage) layoutDesktop(gtx layout.Context) layout.Dimensions {
 		SubTitle:   values.String(values.StrStep2of2),
 		BackButton: pg.backButton,
 		Back: func() {
-			promptToExit(pg.Load, pg.ParentNavigator(), pg.ParentWindow())
+			promptToExit(pg.Load, pg.masterParentID, pg.ParentNavigator(), pg.ParentWindow())
```
Comment (Sirmorrison) [inline ui/page/seedbackup/verify_seed.go:266]: Not needed.

File: ui/page/seedbackup/backup_instructions.go [hunk 2]
```diff
@@ -90,19 +92,19 @@ func (pg *BackupInstructionsPage) HandleUserInteractions() {
...
-func promptToExit(load *load.Load, pageNavigator app.PageNavigator, window app.WindowNavigator) {
+func promptToExit(load *load.Load, masterParentID string, pageNavigator app.PageNavigator, window app.WindowNavigator) {
 	infoModal := modal.NewCustomModal(load).
 		Title(values.String(values.StrExit) + "?").
 		Body(values.String(values.StrSureToExitBackup)).
 		SetNegativeButtonText(values.String(values.StrNo)).
 		SetPositiveButtonText(values.String(values.StrYes)).
 		SetPositiveButtonCallback(func(_ bool, _ *modal.InfoModal) bool {
-			pageNavigator.ClosePagesAfter("Main")
+			pageNavigator.ClosePagesAfter(masterParentID)
```
[the remark this answers is not in the document]
Comment (ukane-philemon) [reply ui/page/seedbackup/backup_instructions.go:107]: Oh, will try this. Thanks for the review.

[the remark this answers is not in the document]
Comment (ukane-philemon) [reply ui/page/seedbackup/backup_instructions.go:107]: Your suggestion closes the current page correctly but I think we should exit the whole backup process if a user cancels at any stage, correct?

[the remark this answers is not in the document]
Comment (Sirmorrison) [reply ui/page/seedbackup/backup_instructions.go:107]: Not exactly... i could feel like i missed something in the previous page and just want to return... this does not necessary mean i want to stop the back up process. The back up process is actually very mandatory.

[the remark this answers is not in the document]
Comment (dreacot) [reply ui/page/seedbackup/backup_instructions.go:107]: if they cancel the backup process at any stage, they need to start afresh, it should remain as that, changing that flow is outside the scope of this PR, an issue can be opened to discuss further on this if needed. this PR should only be fixing the bug outlined in the issue

[the remark this answers is not in the document]
Comment (ukane-philemon) [reply ui/page/seedbackup/backup_instructions.go:107]: > if they cancel the backup process at any stage, they need to start afresh, it should remain as that, changing that flow is outside the scope of this PR

Okay. 

@Sirmorrison, We need `pg.masterParentID` to track the first screen do you have any suggestion to improve on this?

**Review by dreacot — APPROVED** [review]

**Pull request merged**